libs/vscode/nx-cloud-view/src/lib/format-time.ts

```diff
@@ -0,0 +1,37 @@
+export function formatMillis(valueInMilliseconds: number) {
```

Comment:
Can we use `Intl.DateTimeFormat` here?

Reply:
I don't think so... It looks like it's used for dates & times rather than durations. I copied this code over from the cloud app because I want it to be the same as we show in the app.